x-pack/plugins/case/server/client/cases/push.ts

```diff
@@ -73,9 +98,21 @@ export const push = ({
     );
   }
 
+  const { ids, indices }: AlertInfo = theCase?.comments
+    ?.filter(isCommentAlertType)
+    .reduce<AlertInfo>(
+      (acc, comment) => {
+        acc.ids.push(...getAlertIds(comment));
+        acc.indices.add(comment.index);
+        return acc;
+      },
+      { ids: [], indices: new Set<string>() }
+    ) ?? { ids: [], indices: new Set<string>() };
```

Comment:
nit: Could be extracted to a new variable and used in both places

Reply:
👍  I extracted to a helper